content/src/roadmaps/license-tasks/ticket-broker-reseller-registration.md

```diff
@@ -12,7 +12,7 @@ agencyId: "nj-consumer-affairs"
 agencyAdditionalContext: "Regulated Business Section"
 industryId: "retail"
 licenseCertificationClassification: "undefined"
-licenseName: ""
+licenseName: "Ticket Brokers"
 summaryDescriptionMd: "You need to register as a ticket broker if you resell admission tickets to places of entertainment and charge a premium, in addition to the price plus the taxes printed on the tickets."
 ---
```

Comment:
So this one is hidden behind a profile question. That is intentional right? (it isn't called out in AC)

Reply:
Yes that is fine..